### DISCUSSION TITLE
Application getting 'Killed'

### DISCUSSION THREAD

**Discussion by anishatnitt** [idaholab/malamute#111] [Q&A General]
## Bug Description
 Was able to install and run Malamute. All the tests were successful. Whenever I try to run the examples the process starts but eventually fails at some time step

## Steps to Reproduce
Have tried all the examples that come with the package. All of them start and run for a few hours and fail. Have tried with -j 1, -j 2 and the same problem.

## Impact
Have tried in Windows and Linux environment. In both the scenarios its the same result. Config of my Laptop is AMD® Ryzen 5 5600h 6 core, 8GB ram with a dedicated GPU. Tried it on a desktop PC also with config Ryzen 7 3700X 8-Core, 16 GB ram but encountered the same problem.

Looking forward to suggestions from the community. 

Thanks. 

![Screenshot](https://github.com/idaholab/malamute/assets/146836197/147001d7-e10b-437d-b611-5459b34d6846)

**Comment by cticenhour** [comment]
Looking at the memory usage output in the screenshot, I would first guess it being a memory capacity problem coupled with the number of cores being used. Are you running it with more cores than just 1 or 2? 

Pinging @jiangwen84 and @dewenyushu for their input, as this is related to DED.

**Reply by anishatnitt** [reply]
Hi. I have only tried max. 2 cores in my laptop. In the desktop I have tried 1, 2 and 4 cores and all have failed the same way. As I increase the core, the faster the process fails as the RAM usage goes above 85%. When I tried the process with 1 core the RAM usage is considerably less and the program runs for a longer duration but fails eventually. 

P.S: I have tried DED Hollow Cylinder, Cylinder, Melt pool 2D, Melt Pool 3D and others
